client/src/Actions/BalanceActions.js

```diff
@@ -1,32 +1,34 @@
 import axios from 'axios';
-import { URL } from './index';
+
 import * as types from './actionTypes';
 import * as creators from './ActionCreators/BalanceActionCreators';
 
-import { tokenIsValid } from './../Helpers/tokenIsValid'
-import { tokenIsNotValid } from './../Helpers/tokenIsNotValid'
+import { pathObj } from '../Utils/pathVariables';
+import { tokenIsValid } from '../Utils/tokenIsValid'
+import { tokenIsNotValid } from '../Utils/tokenIsNotValid'
```

Comment:
Here it seems that you are importing a dispatch function from `Utils`. This seems inconsistent compared to the other dispatchers that you are importing from `/ActionCreators/`. Is this intentional?

Reply:
Yes, that was intentional. My reason behind is that I would ideally like to reserve the `Actions` and `ActionCreators` folders for everything that directly interfaces with the reducers in one way or another. 

While the utility functions do so as well, they're further abstracted. Since they're usable throughout the entire app, and not just in the Actions, I figured I'd place them in the `Utils` directory to make for easier access. I could place them in a subfolder named something like `tokenUtils` to make it more clear what it pertains to.

Thoughts?

Reply:
Yep, a subfolder sounds more clear. 👍